github/ci/prow/files/jobs/nmstate/kubernetes-nmstate-periodics.yaml

```diff
@@ -0,0 +1,94 @@
+periodics:
+- name: periodic-publish-kubernetes-nmstate-flakefinder-weekly-report
+  interval: 24h
+  decorate: true
+  spec:
+    nodeSelector:
+      type: vm
+      zone: ci
+    containers:
+    - image: index.docker.io/kubevirtci/flakefinder@sha256:1d359b10c91008c2f5a027fc56dbad9d7618361c6f0f8cdc461bddbbced4c2f5
+      env:
+      - name: GOOGLE_APPLICATION_CREDENTIALS
+        value: /etc/gcs/service-account.json
+      command:
+      - "/app/robots/flakefinder/app.binary"
+      args:
+      - --token=/etc/github/oauth
+      - --merged=168h
+      - --report_output_child_path=nmstate/kubernetes-nmstate
+      volumeMounts:
```

Comment:
This is suppose dto report on https://github.com/nmstate/kubernetes-nmstate, right?
You need to set the parameters `org` and `repo` also. Furthermore you need to make sure that the token provided has access to the GitHub repo.